using-mobile-extensions/mobile-core/signal/README.md

```diff
@@ -0,0 +1,211 @@
+{% endtabs %}
+
+## Register the Signal extension
+
+The `registerExtension()` API registers the Signal extension with the MobileCore extension. This API allows the extension to send and receive events to and from the Mobile SDK.
+
+To register the Signal extension, use the following code sample:
+
+{% tabs %}
+{% tab title="Android" %}
+After calling the `setApplication()` method in the `onCreate()` method, register the extension. If the registration was not successful, an `InvalidInitException` is thrown.
+
+#### Java
+
+```java
+public class MobiletApp extends Application {
+@Override
+public void onCreate() {
+super.onCreate();
+     MobileCore.setApplication(this);
+     try {
+         ...........................
+         Signal.registerExtension();
+         ...........................
+     } catch (Exception e) {
+         //Log the exception
+     }
+  }
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### iOS
+
+Register the Signal extension in your app's `didFinishLaunchingWithOptions` function:
+
+**Objective-C**
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [ACPSignal registerExtension];
+  // Override point for customization after application launch.
+  return YES;
+}
+```
+
+**Swift**
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  ..............................
+  ACPSignal.registerExtension()
+  ..............................  
+  // Override point for customization after application launch.
+  return true;
+}
+```
+
+{% endtab %}
+
+{% tab title="React Native" %}
+
+#### JavaScript
+
+```jsx
+import {ACPSignal} from '@adobe/react-native-acpcore';
+
+initSDK() {
+    ACPSignal.registerExtension();
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Version of the Signal extension
+
+The `extensionVersion()` API returns the version of the Signal extension that is registered with the MobileCore extension.
+
+To get the version of the Identity extension, use the following code sample:
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+```java
+String signalExtensionVersion = Signal.extensionVersion();
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### iOS
+
+**Objective-C**
+
+```objectivec
+NSString *signalExtensionVersion = [ACPSignal extensionVersion];
+```
+
+**Swift**
+
+```swift
+var signalExtensionVersion  = ACPSignal.extensionVersion()
+```
+
+{% endtab %}
+
+{% tab title="React Native" %}
+
+#### JavaScript
+
+```jsx
+ACPSignal.extensionVersion().then(signalExtensionVersion => console.log("AdobeExperienceSDK: ACPSignal version: " + signalExtensionVersion));
+```
+
+{% endtab %}
+{% endtabs %}
+
+##Define rules in Adobe Experience Platform Launch
+
+Define the rules in AEP Launch by following the steps in the [Signal extension and Rules engine Integaration document](signals-extension-and-rules-engine-integration)
```

Comment:
- Please replace "AEP Launch" with "Experience Platform Launch"
- Please change "engine" --> "Engine"
- Please lower case "Integration" --> "integration
- please remove "document" from the link text so that it reads "Signal extension and Rules Engine integration"